lib/openstudio/extension/core/os_lib_helper_methods.rb

```diff
@@ -265,7 +265,20 @@ def self.check_upstream_measure_for_arg(runner, arg_name)
           result = measure_step.result.get
           result.stepValues.each do |arg|
             name = arg.name
-            value = arg.valueAsVariant.to_s
+            # check if value, double, int, or bool
+            value_type = arg.variantType.valueDescription
```

Comment:
is this an openstudio change?

Reply:
There was no change in OpenStudio, I'm just making this more robust so it doesn't return everything as a string, which is what was happening before. Then in the measure I had added code to convert back to integer or double in specific use cases. Now that should just work without any special code in the measure.